extensions/3rdparty/LyricWiki/Special_LinksToRedirects.php

```diff
@@ -176,8 +176,17 @@ function execute( $par ){
 					);
 					$idToTitle = array();
 					$ids = array_unique($ids);
-					$queryString = "SELECT page_id, page_namespace, page_title FROM $TABLE_PREFIX"."page WHERE page_id IN (".implode(",", $ids).")";
-					$res2 = $dbr->query($queryString);
+
+					$res2 = $dbr->select(
+						"page",
+						[
+							"page_id",
+							"page_namespace",
+							"page_title"
+						],
+						"page_id IN (".implode(",", $ids).")",
```

Comment:
You can use the following instead:

```php
'page_id' => $ids
```

MediaWiki DB abstraction layer will replace this one (when `$ids` is an array) with `page_id IN (....)`.